iroha2/sys/iroha_data_model/asset/__init__.py

```diff
@@ -11,10 +11,10 @@
                                "iroha_data_model.asset.AssetDefinition"),
                               ("registered_by", "iroha_data_model.account.Id")]
 
-AssetValue = Enum[("Quantity", int), ("BigQuantity", int),
+AssetValue = Enum[("Quantity", int), ("BigQuantity", int), ("Fixed", float),
```

Comment:
Should `Fixed` be represented as float? I think it is better to use also a fixed point arithmetic lib in python.

Reply:
I dont think it matters that much for the first version of python library. We can change it later

Reply:
Ok, let's create an issue for that in python lib

Reply:
Also do you currently do any conversion between Fixed and float? I couldn't find the code, or is it for the later?

Reply:
Hmm.. No i dont. I would do it later in pr with crypto primitives

Reply:
Ok, maybe create an issue for that to :)